content/hardware/build_instructions.md

```diff
@@ -783,17 +778,17 @@ In this section, we will mount the D435i Realsense camera to the front cover of
 
 **Steps**
 
-1. Plug the 1ft USB 3.0 cable into the D435i camera. Note that we are not using the USB cable that comes with the camera because it is excessively long.
+1. Plug the 1ft USB 3.0 cable into the D435i camera. Note that we are not using the USB cable that comes with the camera because it is excessively long. If you are using the D455 camera and a Xavier NX, use a right-angled USB-C connector between the USB cable and the camera so that the cable does not block the second battery.
```

Comment:
Have we tested that:
1. This USB-C right angle adapter fits?
2. When using the adapter, we can still get RGB-D images at 30Hz?

Reply:
Yes, both have been tested and the adapter works fine.